packit/upstream.py

```diff
@@ -426,7 +426,8 @@ def get_archive_extension(self, archive_basename: str, version: str) -> str:
             # can be different from Spec version. Replace it to ensure proper match.
             base = base.replace(self.specfile.get_version(), version)
             if base.startswith(archive_basename):
-                return base[len(archive_basename):]
```

Comment:
black changed this into `...[len(arch...): ]` (see the space), which obviously is not pep8, hence I had to rewrite it